apps/mark-scan/backend/src/electrical_testing/background.ts

```diff
@@ -28,8 +28,8 @@ import { AudioOutput, setAudioOutput } from '../audio/outputs';
 const CARD_READ_INTERVAL_SECONDS = 5;
 const PAPER_HANDLER_POLL_INTERVAL_MS = 250;
 const PAPER_LOAD_LOG_INTERVAL_MS = 5000;
-const HEADPHONE_OUTPUT_DURATION_SECONDS = 60;
-const SPEAKER_OUTPUT_DURATION_SECONDS = 5;
+const HEADPHONE_OUTPUT_DURATION_SECONDS = 50;
+const SPEAKER_OUTPUT_DURATION_SECONDS = 10;
```

Comment:
`apps/mark-scan` doesn't have the same sound utilities used in `apps/scan` to play on an interval. To avoid a constant chime I edited the sound file to be 5 seconds long total, with 1.5s of the "success" chime and 3.5s of silence.

To ensure at least one sound is played through the speaker every cycle I increased SPEAKER_OUTPUT_DURATION_SECONDS to be >5s.